tests/ra_aid/tools/test_handle_user_defined_test_cmd_execution.py

```
@@ -0,0 +1,184 @@
+
+def test_execute_test_command_function():
+    """Test the execute_test_command function."""
+    config = {"test_cmd": "test"}
+    with patch("ra_aid.tools.handle_user_defined_test_cmd_execution.TestCommandExecutor") as mock_executor_class:
+        mock_executor = Mock()
+        mock_executor.execute.return_value = (True, "new prompt", True, 1)
+        mock_executor_class.return_value = mock_executor
+        
+        result = execute_test_command(config, "prompt", auto_test=True)
+        assert result == (True, "new prompt", True, 1)
+        mock_executor_class.assert_called_once_with(config, "prompt", 0, True)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)